public/wp-content/themes/ColoredCow/header.php

```diff
@@ -7,6 +7,25 @@
     <meta name="viewport" content="initial-scale=1">
     <title><?php echo bloginfo('name'); ?></title>
     <?php wp_head(); ?>
+
+	<nav class=" navbar fixed-top navbar-toggleable-md navbar-inverse bg-inverse">				
+		<button class="navbar-toggler navbar-toggler-right justify-content-end" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+			<span class="navbar-toggler-icon"></span>
+		</button>
+		<a class="navbar-brand" href="#">
+			<img src="<?php echo get_template_directory_uri(); ?>/dist/img/logo.png" width="180" height="45" class="d-inline-block align-top" alt="">
+		</a>
+		<div class="collapse navbar-collapse justify-content-end" id="navbarSupportedContent">
+			<ul class="navbar-nav">
+				<li class="nav-item active">
+					<a role="button" class="btn btn-outline-secondary"  href="http://localhost/ph2_RSVP/public/wp/wp-admin" style="font-family: 'Oswald'; font-size: 17px;">Admin Login
+						<span class="sr-only">(current)</span>
+					</a>
+				</li>&nbsp;
```

Comment:
Does this &nbsp; has any specific purpose here? If not, please remove it. Make your code cleaner.